x-pack/plugins/cloud_security_posture/public/pages/findings/findings.tsx

```diff
@@ -5,25 +5,27 @@
  * 2.0.
  */
 import React from 'react';
-import type { EuiPageHeaderProps } from '@elastic/eui';
+// eslint-disable-next-line @kbn/eslint/module_migration
+import styled from 'styled-components';
 import { useKubebeatDataView } from '../../common/api/use_kubebeat_data_view';
 import { allNavigationItems } from '../../common/navigation/constants';
 import { useCspBreadcrumbs } from '../../common/navigation/use_csp_breadcrumbs';
 import { FindingsContainer } from './findings_container';
 import { CspPageTemplate } from '../../components/csp_page_template';
-import { FINDINGS } from './translations';
 
-const pageHeader: EuiPageHeaderProps = {
-  pageTitle: FINDINGS,
-};
+const FindingsPageTemplate = styled(CspPageTemplate)`
```

Comment:
This doesn't seem to do anything, when I remove it and use `CspPageTemplate` directly the page looks the same. I think that removing the `pageHeader` prop removes the header, so no need for this hack